src/python/dxpy/scripts/dx.py

```diff
@@ -4592,6 +4592,7 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
 parser_update_project.add_argument('--containsPHI', choices=["true"],
                                    help="Flag to tell if project contains PHI")
 parser_update_project.add_argument('--bill-to', help="Update the user or org ID of the billing account", type=str)
+parser_update_project.add_argument('--allowed-executables', help='Executable ID(s) this project is allowed to run.  This operation overrides any existing list of executables.', type=str, nargs="+")
```

Comment:
Maybe add here a note: "Passing null removes the list" or similar. Empty list means no executables are allowed to run?